examples/MagicWeatherCompose/app/src/main/java/com/revenuecat/sample/utils/EntitlementInfosExtensions.kt

```diff
@@ -0,0 +1,7 @@
+package com.revenuecat.sample.utils
+
+import com.revenuecat.purchases.EntitlementInfos
+
+fun EntitlementInfos.hasActiveEntitlements(): Boolean {
+    return this.active.isNotEmpty()
+}
```

Comment:
We could add another one here that gets the active entitlements by doing `active.keys`

Reply:
Well, I think that makes sense, but we are currently not displaying the active entitlements anywhere, only whether a user has any active entitlements or not. 

We could add a place where it displays the active entitlements, and checks that it has the correct one instead, as we do in the current magic weather, this just seemed a simpler approach, but I'm ok changing it. Thoughts? @vegaro

Reply:
Talked with @vegaro. Done in https://github.com/RevenueCat/purchases-android/pull/1056/commits/5ab1901049e9a511a722d95d12e2ab62b3800b36